src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -10,10 +10,12 @@ import { id as taskId } from '@/components/nodes/task';
 import { id as callActivityId } from '@/components/nodes/callActivity';
 import { id as manualTaskId } from '@/components/nodes/manualTask';
 import { id as scriptTaskId } from '@/components/nodes/scriptTask';
+import portsConfig from '@/mixins/portsConfig';
+import { getAnchorCoordinates } from '@/snapToAnchor';
```

Comment:
Do we need braces if it's a default export? Doesn't need fixing now, minor point. :)